apps/web/cypress/global.d.ts

```diff
@@ -43,6 +43,8 @@ declare namespace Cypress {
      */
     inviteUser(email: string): Chainable<Response>;
 
+    loginWithGithub(): Chainable<any>;
```

Comment:
```suggestion
    loginWithGitHub(): Chainable<any>;
```
To be in line with their branding name and the camel casing convention.